src/empire/__init__.py

```diff
@@ -1,18 +0,0 @@
-import sys
-
-if sys.version_info[:2] >= (3, 8):
-    # TODO: Import directly (no need for conditional) when `python_requires = >= 3.8`
-    from importlib.metadata import PackageNotFoundError, version  # pragma: no cover
-else:
-    from importlib_metadata import PackageNotFoundError, version  # pragma: no cover
-
-try:
```

Comment:
any idea on what all of this does?

Reply:
Defines dist_name and version of our package. We don't really need this anyway